deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -57,6 +57,9 @@ spec:
         command:
           - "/otelcontribcol"
           - "--config=/conf/traces.otelcol.conf.yaml"
+          {{- if eq .Values.otelcol.metrics_enabled false }}
+          - "--metrics-level=none"
+          {{- end }}
```

Comment:
What are the other metrics levels?

Reply:
I didn't find anything about levels description but levels itself are described [here](https://github.com/open-telemetry/opentelemetry-collector/blob/e7e66939263ccbd99c9ac9352960c263d464263c/internal/collector/telemetry/telemetry.go#L48) Looks like to get information what's under which level you have to go thru the code.